Installation/include/CGAL/license/generate_files.cmake

```diff
@@ -6,6 +6,9 @@ foreach(LINE ${package_list})
   string(TOUPPER ${PACKAGE_CODE_NAME} PACKAGE_CODE_NAME)
   string(REPLACE "${PACKAGE_NAME}" "" PACKAGE_PRETTY_NAME ${LINE})
   string(STRIP "${PACKAGE_PRETTY_NAME}" PACKAGE_PRETTY_NAME)
+  if(NOT PACKAGE_PRETTY_NAME)
+    set(PACKAGE_PRETTY_NAME ${PACKAGE_NAME})
+  endif()
```

Comment:
@danston Have you check if that code has indeed the same behavior of the previous version?

Reply:
Yes, it did not update any other license except for the following 4 packages: `Classification`, `Orthree`, `Three`, and `Triangulation_on_sphere_2`. The `Orthree` and `Triangulation_on_sphere_2` are modified even when running the original master version. The `Three` and `Classification`, the empty space in the license header is now substituted by the correct name.

Reply:
The question is: should I add those license modifications to this PR as well?

Reply:
> The question is: should I add those license modifications to this PR as well?

That is too late: I have already merged it. :smiley: